### PR TITLE
Build Log sending by email for the build

### LINKED ISSUE #98: Generator branch iso gets drafted sometime
Sometime iso released by generator branch of meilix get drafted rather than getting into release.
And email doesn't send to the user unless until there is a deployment of iso.
https://github.com/fossasia/meilix/pull/94

### PR DESCRIPTION
**Pull request by xeon-zolt**
### Short description
I have:
- [x] There is a corresponding issue for this pull request.
- [x] Mentioned the Issue number in the pull request commit message 

Fixes #98 so that user can receive build log via email with ISO

- [x] sending email with log text attached
- [x] sending log after deployment
- [x] sending log in case of build failure
### For the reviewers
I have:
- [ ] Reviewed this pull request by an authorized contributor.
- [ ] The reviewer is assigned to the pull request.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -8,10 +8,10 @@ git:
 sudo: required
   
 script:
-  - ./build1.sh
+  - ./build1.sh |& tee log.txt
  
 before_deploy:
-  - sudo rm meilix-original.iso
+  - sudo rm meilix-original.iso
```
Comment (abishekvashok) [inline .travis.yml]: trailing white space

**Comment by tabesin** [comment]
I assume we are ready for merge. Anyway, I need some clarity about the xeon-zolt hard links before we can merge this to the fossasia repo. We merged that hard link before, I am just wondering if that will work on the fossasia repo.

**Comment by xeon-zolt** [comment]
currently, the deployments are done in my repo due to too many beta builds being collected and itss a mess in release section so I am going to make a pull to remove those hard links with help of hosting on transfer.sh #122

**Comment by tabesin** [comment]
ok let's ship it for now.

**Pull request merged**